sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -601,7 +602,8 @@ def test_radius_neighbors_classifier_zero_distance():
                                                       weights=weights,
                                                       algorithm=algorithm)
             clf.fit(X, y)
-            assert_array_equal(correct_labels1, clf.predict(z1))
+            assert_array_equal(correct_labels1,
+                               assert_no_warnings(clf.predict, z1))
```

Comment:
As explained in #19396, no need to assert that every line of our code does not issue a warning.